packages/rabbitmq/src/amqp/utils.ts

```diff
@@ -0,0 +1,20 @@
+export function matchesRoutingKey(
```

Comment:
i would love to see some test coverage added. 
E2E is also a plus as i'd like to assert that this logic works as expected

Reply:
will add some later today

Reply:
@Andrius0124 Alright, just let me know and I'll review/merge if needed

Reply:
@underfisk  Updated. Let me know if I missed anything